include/picongpu/particles/particleToGrid/derivedAttributes/BoundElectronDensity.def

```diff
@@ -0,0 +1,114 @@
+/* Copyright 2015-2018 Axel Huebl
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+#include "picongpu/traits/SIBaseUnits.hpp"
+#include "picongpu/particles/traits/SpeciesEligibleForSolver.hpp"
+
+#include <pmacc/traits/HasIdentifiers.hpp>
+
+#include <vector>
+
+
+namespace picongpu
+{
+namespace particles
+{
+namespace particleToGrid
+{
+namespace derivedAttributes
+{
+    /** Density of Bound Electrons Operation for Particle to Grid Projections
```

Comment:
OPTIONAL:
This sentence sounds not correct - is there a better wording?

Reply:
that's how we document all of them - not gentle for long names but okish